internal/server/redis.go

```diff
@@ -76,6 +93,15 @@ func (sess *redisSession) write(s *server, msg string) {
 	}
 }
 
+func isValidKey(key string) bool {
+	for _, r := range illegalKeyRunes {
+		if strings.ContainsRune(key, r) {
+			return false
+		}
+	}
```

Comment:
This could eventually get expensive if we have a bunch of illegal runes but is probably fine for now.